clr-k8s-examples/setup_system.sh

```diff
@@ -6,9 +6,72 @@ set -o nounset
 ADD_NO_PROXY="10.244.0.0/16,10.96.0.0/12"
 ADD_NO_PROXY+=",$(hostname -I | sed 's/[[:space:]]/,/g')"
 
-#Install kubernetes and crio
-sudo -E swupd update
-sudo -E swupd bundle-add cloud-native-basic storage-utils
+function clear_setup()
+{
+  #Install kubernetes and crio
+  sudo -E swupd update
+  sudo -E swupd bundle-add cloud-native-basic storage-utils shells
+  echo "source <(kubectl completion bash)" >> $HOME/.bashrc
+}
+
+function ubuntu_setup()
+{
+  #Given the vagrant box we are using we need to really really do this hack for networking to work.
+  # Check https://github.com/lavabit/robox/issues/35
+  sudo sed -i "/nameservers/d" /etc/netplan/01-netcfg.yaml
+  sudo sed -i "/addresses/d" /etc/netplan/01-netcfg.yaml
+  sudo sed -i "s/DNS=.*/DNS=/g" /etc/systemd/resolved.conf
+  sudo sed -i "s/DNSSEC=.*/DNSSEC=no/g" /etc/systemd/resolved.conf
+  sudo netplan generate
+  sudo netplan apply
+  sudo systemctl restart systemd-networkd.service systemd-resolved.service
+
```

Comment:
How much of the above is box specific vs server install of ubuntu 1804?
Should we split up the script per OS with maintainable duplication for sake of simplicity?

Reply:
That way end users dont have to deal with quirks introduced by us supporting multiple vagrant environments.

Reply:
Test without Vagrant and see where these scripts fails

Reply:
Capture change in a README update